test-server.js

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var express = require('express');
+var app = express();
+var adapter_manager = require('./adapters.js');
+var bodyParser = require('body-parser');
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+const ROOT = '/things';
```

Comment:
Note that currently this clashes with app.js which also tries to respond to HTTP requests to this path. This is fine for a test but we need to decide where this logic lives and it should probably eventually live in an Express routes/ directory. Also I don't think it should directly call the AdapterManager to get a list of Things, that should probably be handled by a separate ThingManager which gets a list of Things from a database and calls the AdapterManager to look for new Things or get or set properties etc.